packages/terra-dialog/src/terra-dev-site/test/dialog/DefaultDialog.test.scss

```diff
@@ -0,0 +1,6 @@
+:local {
```

Comment:
Shouldn't these sass files be `name.module.scss`? @yuderekyu @bjankord

Reply:
For consistency sake, yeah they should. We use the `name.module.scss` format to make sure that CSS modules are detected and applied when the scss files are used in tools like create-react-app, rails/webpacker, codesandbox.io. Since it is unlikely people will use these test files in those tools, its not as big of a concern but I think for consistency, we should go with the  `name.module.scss` format for any file that is intended to be used as a css module.

Reply:
To maintain consistency i have updated the scss file names to follow the `name.module.scss` format.